### PR TITLE
Polymer expects the dir attribute to be set on the documentElement

### PR DESCRIPTION
**Pull request by awikkerink**
See this for context: https://github.com/Polymer/polymer/blob/ee1d71f5f896ee57a0ee6d1f377ae6caa76e556b/lib/mixins/dir-mixin.html#L33

### CODE REVIEW

File: d2l-demo-template.html [hunk 1]
```diff
@@ -229,7 +229,7 @@ <h1 class="d2l-heading-4" style="margin: 0;">[[title]]</h1>
 				};
 
 				if (fixtures.length === 0) {
-					update(document.body);
+					update(document.documentElement);
 				} else {
 					for (var i = 0; i < fixtures.length; i++) {
 						update(fixtures[i]);
```
Comment (awikkerink) [inline d2l-demo-template.html]: I'm wondering why this was set in the first place? Should this case be removed since this won't work anyways?

Comment (dbatiste) [reply d2l-demo-template.html]: There's no requirement that the template contains fixtures, so I think this conditional is just set to handle the case where someone is using the template without fixtures.  The expected use-case is certainly to have fixtures, and in such case, the demo-template code just tries to apply dir to the fixture containers (which is technically valid... Polymer aside).  Anyway, there might be one or two places that don't use a fixture in the demo... maybe something like floating workflow buttons, but I don't recall.  I would not worry too much about removing this case.

Comment (awikkerink) [reply d2l-demo-template.html]: But with the way Polymer 2 is currently written, rtl styling will not be applied to the fixtures correctly. The polyfill only handles the case where dir=rtl is set at the `documentElement` level

Comment (dbatiste) [reply d2l-demo-template.html]: So we could just roll with that to address this.  The original idea was to enable toggling RTL for the thing being demo'd without the whole page being flipped RTL, but it's not critical that it be done that way.

Eventually I suspect the demo-template may be ditched.  All though the iron demo pages don't provide us with all the switches and such, we've been adopting that more for demos.

**Comment by awikkerink** [comment]
@dlockhart @dbatiste looks good?

I don't have permission to merge btw

**Comment by awikkerink** [comment]
Could you merge/release?

**Comment by dlockhart** [comment]
This is mostly @dbatiste's baby, so once he's had a look for sure.

**Pull request merged**